### PR TITLE
Update save to eeprom functionality

### PR DESCRIPTION
**Pull request by ranechita**
Signed-off-by: Ramona Alexandra Nechita <ramona.nechita@analog.com>

# Description

When running the production tests, at the end of the calibration procedure a window is prompted to introduce the serial number of the device and then eeprom is written accordingly. All functions related to the eeprom are moved to a separate file. It also includes minor fixes.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

**Test Configuration**:
* Hardware: ADRV9361-ZC702
* OS: Ubuntu 20.04

# Documentation

If this is a new feature or example please mention or link any documentation. All new hardware interface classes require documentation.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have signed off all commits and they contain "Signed-off by: <insert name>"
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: test/eeprom.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    if ssh_stderr.channel.recv_exit_status() != 0:
+        raise paramiko.SSHException("find_eeprom command did not execute properly")
+
+    this_day = date.today()
+    if this_day.year <= 1995:
+        pytest.skip(
+            "System date and time not in order. PLease connect the device to internet to update."
+        )
+
+    sn = popup_txt()
```
Comment (tfcollins) [inline test/eeprom.py:32]: Can we add a check here to make sure the input is reasonable

File: test/scpi.py [hunk 2]
```diff
@@ -185,7 +116,6 @@ def dcxo_calibrate(classname, iio_uri):
 
     sdr._set_iio_dev_attr("dcxo_tune_coarse", coarse, sdr._ctrl)
     sdr._set_iio_dev_attr("dcxo_tune_fine", fine, sdr._ctrl)
-
     while 1:
         current_frq = get_freq(instr)
```
Comment (tfcollins) [inline test/scpi.py:120]: Any scenario when this would never exit?

File: test/eeprom.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
+import tkinter as tk
+from datetime import date, datetime
+from tkinter import Button, Entry, Label
```
Comment (tfcollins) [inline test/eeprom.py:3]: Will need a separate requirements.txt file (maybe requirements_prod_test.txt). TK doesn't always ship with python

File: test/eeprom.py (hunk 1 above)
Comment (ranechita) [reply test/eeprom.py:32]: I added a check in the popup_txt() function, right before returning the value

File: test/scpi.py (hunk 2 above)
Comment (ranechita) [reply test/scpi.py:120]: I don't think there is any

File: test/eeprom.py (hunk 3 above)
Comment (ranechita) [reply test/eeprom.py:3]: I created a new commit to add that

**Pull request merged**